python/remotivelabs-broker/docker-build.sh

```diff
@@ -1,21 +1,21 @@
 #!/bin/bash
-
+#
+# Generates code from proto files, builds package and generates documentation
+#
 set -e
 
-# Generates code from proto files, builds package and generates documentation
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+TAG=remotivelabs/python-api-build-image
 
-docker build -t remotivelabs/python-api-build-image -f docker/Dockerfile .
+docker build -t ${TAG} -f ${SCRIPT_DIR}/docker/Dockerfile ${SCRIPT_DIR}
```

Comment:
From your personal shell linter ;)

[SC2086](https://www.shellcheck.net/wiki/SC2086)
[SC2046](https://www.shellcheck.net/wiki/SC2046)

Reply:
okok. Do we have a shell linter?

Reply:
Nop